package.json

```diff
@@ -13,6 +13,10 @@
   "activationEvents": [
     "*"
   ],
+  "engines":{
+    "vscode": "*",
+    "azdata": "*"
+  },
```

Comment:
Why was this added back in?

Reply:
Turned out packaging requires these engines